### DISCUSSION TITLE
Syslog pipeline timezone change

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#12400] [Unsupported Versions]
Hi!

I try to change the timezone for my Syslog pipeline like this

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/81571847/751dd48e-5f7b-4c13-aece-d4cf631ebe35)

and the pipeline stops to work. Why? Isn't the syntax correct? Thanks in advance!

Cheers, Ben

**Comment by cm-ops** [comment]
Setting looks correct, do you get an error in the log?

**Reply by ben-sec** [reply]
So far I couldn't find anything. It should be in elasticsearch.log, shouldn't it? Master or search nodes?

**Reply by cm-ops** [reply]
Check on the search node. Also, check the syslog pipeline stats - `sudo so-elasticsearch-pipeline-stats syslog` for any errors.

**Reply by ben-sec** [reply]
I will park this problem, since we plan to switch to SO 2.4 shortly and then we will no longer use UTC in SO. Thank you anyway!
